lib/http2.c

```diff
@@ -2011,6 +2012,20 @@ static ssize_t h2_submit(struct stream_ctx **pstream,
     }
   }
 
+  body = (const char *)buf + nwritten;
+  bodylen = len - nwritten;
+
+  if(bodylen > 0) {
```

Comment:
I think this should do `if(bodylen)` since anything non-zero will be larger than zero when the type is unsigned.